### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [openfoodfacts/openfoodfacts-server#6027] [General]
Would a Mastodon account for this project be possible?

**Comment by VaiTon** [comment]
@Lukas2112 As the android app or as the entire project?

cc @stephanegigandet @teolemon

**Reply by linux-lukas** [reply]
An account on Mastodon - or generally in a part of the Fediverse - for the project. 

From my perspective, the following things would be possible via such a privacy-friendly channel: 

- Share project information and project announcements 
- interaction with the Fediverse community

positive side effects would be
- Mastodon & as a privacy-friendly social media would become better known.
- The project would certainly gain publicity

**Reply by teolemon** [reply]
We have one https://mastodon.social/@openfoodfacts
